src/students.ts

```diff
@@ -133,8 +133,10 @@ function countCourses(student: Student): number {
  */
 function removeCourseFromStudent(student: Student, course: string): Student {
   // write your code here...
-
-  return student;
+  return {
+    ...student,
+    courses: student.courses.filter((c) => c !== course),
+  };
```

Comment:
This is clever, but it would be preferable to do this:

```ts
student.courses = student.courses.filter((c) => c !== course);
return student;
```
This way we don't break the reference to the original object